cli/types/cypress.d.ts

```diff
@@ -281,6 +281,11 @@ declare namespace Cypress {
     specType?: CypressSpecType
   }
 
+  /**
+   * List of spec titlePaths (parent suite names + test name)
```

Comment:
```suggestion
   * List of full title path (parent suite names + test name) of the Cloud tests that failed that need debugged.
```

Reply:
How will this work with runAll mode? Assume the long-term goal will be to move that out of experimental, should/will the cloud data return the spec name as well?